### PR TITLE
Update preview_domain() to include value names and descriptions

### LINKED ISSUE #62: Add more information about the attributes
GDC provides the meaning of columns, we could potentially include this information in `preview_domains()`.

### PR DESCRIPTION
**Pull request by roquelopez**
Fix #62.

### CODE REVIEW

File: bdikit/api.py [hunk 1]
```diff
@@ -419,75 +419,61 @@ def preview_value_mappings(
...
-    target: Union[str, pd.DataFrame] = "gdc",
+def preview_domain(
+    column: str,
+    dataset: Union[str, pd.DataFrame] = "gdc",
     limit: Optional[int] = None,
 ) -> pd.DataFrame:
     """
-    Preview the domain (set of unique values) of the given columns in the source and target
-    dataset (or target data dictionary).
+    Preview the domain, e.g. set of unique values, column description and value description
```
Comment (aecio) [inline bdikit/api.py:428]: minor typo: `i.e.` instead of `e.g.`

File: bdikit/api.py [hunk 2]
```diff
@@ -419,75 +419,61 @@ def preview_value_mappings(
...
+    (if applicable) of the given column of the source or target dataset.
 
     Args:
-        dataset (pd.DataFrame): The source dataset containing the columns to preview.
-        column_mapping (Tuple[str, str]): The mapping between the source and target columns.
-            The first and second positions should contain the names of the source and target
-            columns respectively.
-        target (Union[str, pd.DataFrame], optional): The target dataset or standard vocabulary name.
-            If a string is provided and it is equal to "gdc", the target domain will be retrieved
+        column(str): The column name to show the preview domain about.
```
Comment (aecio) [inline bdikit/api.py:432]: nit: "The column name to show the domain."

**Review by aecio — COMMENTED** [review]
The new implementation looks good to me! As we discussed, it would be a little more consistent with other functions if the argument order was inverted so that the dataset comes first. The function call would look like this:
```py
bdi.preview_domain(dataset, column="Histologic_type")
# or
bdi.preview_domain(dataset="gdc", column="primary_diagnosis")
```
Instead of:
```py
bdi.preview_domain(column="Histologic_type", dataset=dataset)
```

**Comment by roquelopez** [comment]
Thanks Aecio! I changed the order of the params (I had it in a local commit). I also addressed your other comments.

**Comment by roquelopez** [comment]
I added a version constraint to `conllu` library (`flair`'s dependency). It was updated today and there were some errors when importing `flair`.

**Pull request merged**